// try and catch 

try{
   
    addalert("Hello");   // Our code we want to check

}

catch(err){  // error
    console.log(err);
}